cmd/juju/cloud/regions.go

```diff
@@ -133,21 +133,20 @@ func (c *listRegionsCommand) Run(ctxt *cmd.Context) error {
 	var returnErr error
 	if c.Client {
 		if err := c.findLocalRegions(ctxt); err != nil {
-			ctxt.Warningf("%v", err)
+			ctxt.Infof("ERROR %v", err)
```

Comment:
These ones should still be a warning?
The command still completes and shows some info, it's just that not everything could be retrieved and those things that couldn't are warned about.